src/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>9</LangVersion>
     <Nullable>enable</Nullable>
-    <UnityVersion>2019.4.21f1</UnityVersion>
+    <UnityVersion>2019.4.19f1</UnityVersion>

```

Comment:
Hm..

Reply:
Only 22f1 shows up for install now for me and I don't have 21f1.

I'd revert this before mering.